src/api/notes/index.ts

```diff
@@ -23,11 +23,18 @@ export interface Note {
 }
 
 export const getNote = async (noteId: string): Promise<Note> => {
+  let response;
   // The "-get" suffix is necessary, because in our mock api (filesystem) the note id might already be a folder.
   // TODO: [mrdrogdrog] replace -get with actual api route as soon as api backend is ready.
-  const response = await fetch(getApiUrl() + `/notes/${ noteId }-get`, {
-    ...defaultFetchConfig
-  })
+  if (process.env.NODE_ENV === 'development') {
+    response = await fetch(getApiUrl() + `/notes/${ noteId }-get`, {
+      ...defaultFetchConfig
+    })
+  } else {
+    response = await fetch(getApiUrl() + `/notes/${ noteId }`, {
+      ...defaultFetchConfig
+    })
+  }
```

Comment:
Couldn't this be shortened by usage of a ternary operator?

Reply:
Something like this?
```
const response = await fetch(getApiUrl() + `/notes/${ noteId }${process.env.NODE_ENV === 'development' ? '-get' : ''}`, {
      ...defaultFetchConfig
    })
```

Reply:
Changed to the mentioned solution

Reply:
imho this line is much harder to read now. Could you please merge the getApiURL into the string template and extract the suffix into a var?

Reply:
Also: do you need the spread syntax here?

Reply:
The spread syntax isn't needed if the `{ }` around it get removed.

Reply:
I think it's best that this is kept as is. We do it in all fetch calls and it makes it easy to later add extra option if we need them.